modules/aardvarkBidAdapter.js

```diff
@@ -0,0 +1,154 @@
+const SYNC_ENDPOINT = 'sync.rtk.io';
+const AARDVARK_TTL = 300;
+const AARDVARK_CURRENCY = 'USD';
+
+var hasSynced = false;
+
+export function resetUserSync() {
+  hasSynced = false;
+}
+
+export const spec = {
+  code: BIDDER_CODE,
+
+  isBidRequestValid: function(bid) {
+    return ((typeof bid.params.ai === 'string') && !!bid.params.ai.length &&
+        (typeof bid.params.sc === 'string') && !!bid.params.sc.length);
+  },
+
+  buildRequests: function(validBidRequests, bidderRequest) {
+    var auctionCodes = [];
+    var requests = [];
+    var requestsMap = {};
+    var referer = utils.getTopWindowUrl();
+    var pageCategories = [];
+
+    if (window.rtkcategories && Array.isArray(window.rtkcategories)) {
+      pageCategories = window.rtkcategories;
+    }
+
+    utils._each(validBidRequests, function(b) {
+      var rMap = requestsMap[b.params.ai];
+      if (!rMap) {
+        rMap = {
+          shortCodes: [],
+          payload: {
+            version: 1,
+            jsonp: false,
+            rtkreferer: referer
+          },
+          endpoint: DEFAULT_ENDPOINT
+        };
+
+        if (pageCategories && pageCategories.length) {
+          rMap.payload.categories = pageCategories.slice(0);
+        }
+
+        if (b.params.categories && b.params.categories.length) {
+          rMap.payload.categories = rMap.payload.categories || []
+          utils._each(b.params.categories, function(cat) {
+            rMap.payload.categories.push(cat);
+          });
+        }
+
+        if (bidderRequest && bidderRequest.gdprConsent) {
+          rMap.payload.gdpr = bidderRequest.gdprConsent.gdprApplies;
+          rMap.payload.consent = bidderRequest.gdprConsent.consentString;
+        }
+
+        requestsMap[b.params.ai] = rMap;
+        auctionCodes.push(b.params.ai);
+      }
+
+      rMap.shortCodes.push(b.params.sc);
+      rMap.payload[b.params.sc] = b.bidId;
+
+      if ((typeof b.params.host === 'string') && b.params.host.length &&
+          (b.params.host !== rMap.endpoint)) {
+        rMap.endpoint = b.params.host;
+      }
+    });
+
+    utils._each(auctionCodes, function(auctionId) {
+      var req = requestsMap[auctionId];
+      requests.push({
+        method: 'GET',
+        url: `//${req.endpoint}/${auctionId}/${req.shortCodes.join('_')}/aardvark`,
+        data: req.payload,
+        bidderRequest
+      });
+    });
+
+    return requests;
+  },
+
+  interpretResponse: function(serverResponse, bidRequest) {
+    var bidResponses = [];
+    utils._each(serverResponse.body, function(rawBid) {
+      if (!rawBid.cpm) {
+        return;
+      }
+
+      var bidResponse = {
+        requestId: rawBid.cid,
+        cpm: rawBid.cpm,
+        width: rawBid.width || 0,
+        height: rawBid.height || 0,
+        currency: rawBid.currency ? rawBid.currency : AARDVARK_CURRENCY,
+        netRevenue: rawBid.netRevenue ? rawBid.netRevenue : true,
+        ttl: rawBid.ttl ? rawBid.ttl : AARDVARK_TTL,
+        exchange: rawBid.ex,
```

Comment:
Can remove this `exchange` field from the `bidResponse` object.  This is not a current field.